src/submission/forms.py

```diff
@@ -1,7 +1,11 @@
 from django import forms
 from django.forms import ModelForm
 
-from core import models as core_models, logic as core_logic
+from core import (
```

Comment:
Like how you've done this import. Very easy to read

Reply:
👍